### ISSUE TITLE
output-exponent-marker in table results in undefined control sequence

### ISSUE THREAD

**Issue by mathsen** [josephwright/siunitx#578]
Hello,

I have an "old" code (just 2 months old), which I try to compile with an updated texlive version.
I also installed texlive manually and updated siunitx to the current master branch of the repo - with no effect.
Trying to compile a code like:

````
\documentclass[10pt,a4paper]{scrreprt}
\usepackage{amsmath}
\usepackage{booktabs}
\usepackage{siunitx}

\begin{document}

Test table is here:

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{
\sisetup{
% commenting out this line results in a working compilation
output-exponent-marker=\ensuremath{\mathrm{e}},
}

\begin{table}[h!t]
\centering
% 
\begin{tabular}{S}
	\toprule
	% 
	{$\Gamma$} \\
	\midrule
	%
	2.7893186804e-05  \\ 
	8.8016369990e-07  \\
	2.7553354750e-08  \\
	8.6104233593e-09  \\
	% 
	\bottomrule
\end{tabular}
\end{table}
}


\end{document}
````

With pdflatex or lualatex results in errors like:

````
! Undefined control sequence.\GenericError ...#4 \errhelp \@err@ ... 8
````

The problematic line is the option
````
output-exponent-marker=\ensuremath{\mathrm{e}},
````
removing this option results in a working compilation.

Maybe someone can help me with this problem?

Thanks and greetings

Mathse

**Comment by josephwright** [comment]
Fixed by b87d51f33afd11674373439133f34fb61c41764c

**Comment by mathsen** [comment]
@josephwright Thank you so much for this quick fix! I just tested the new version and everything works fine, as expected.
So thanks again for the great work and this wonderful package.
